app/assets/javascripts/application.js

```diff
@@ -17,7 +17,15 @@
 //= require d3
 //= require react
 //= require react_ujs
+//= require highcharts
+//= require highcharts/highcharts-more
+//= require highcharts/modules/exporting
+//  require highcharts/modules/heatmap
```

Comment:
Note het missen van de `=` hier. Het lijkt alsof hij de heatmap niet goed wil laden, of dat hij het te laat laad oid. Gek genoeg, als ik m hier include, en kijk in de source:

![mentor_overview 2017-12-14 11-00-05](https://user-images.githubusercontent.com/617574/33986490-fb7ada7c-e0bd-11e7-9a1f-757059c5e4b0.png)

En die file bestaat, is niet leeg, en is (voor zover ik kan zien) dezelfde versie als degene die ik later include (waarmee het wel werkt):

```haml
%script{:src => "https://code.highcharts.com/modules/heatmap.js"}
```

Maar eigenlijk is dat brak. Heb jij nog ideeen waarom het fout gaat? De error die ik krijg is de volgende:

```javascript
highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:114 Uncaught Error: Highcharts error #17: www.highcharts.com/errors/17
    at Object.H.error (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:114:23)
    at H.Chart.initSeries (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:19951:23)
    at http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:21632:27
    at Array.forEach (<anonymous>)
    at H.each (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:1763:67)
    at H.Chart.firstRender (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:21631:17)
    at H.Chart.init (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:19934:23)
    at H.Chart.<anonymous> (http://vsv.io/assets/highcharts/modules/data.self-93f2bc507f1dd304019ae3bef25c76b6b3f3928c2d55e7914f70ee0243d69c88.js?body=1:1591:25)
    at H.Chart.obj.(anonymous function) [as init] (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:937:28)
    at H.Chart.getArgs (http://vsv.io/assets/highcharts.self-eca0546ad4211db8b49e8fc3078bdc8ac58cf0ddad43c013a79ba2862f8fc9b3.js?body=1:19794:22)
```

Reply:
fixed, was een probleem met ublock